statsd/noop_test.go

```diff
@@ -0,0 +1,63 @@
+package statsd
+
+import (
+	"testing"
+	"time"
+)
+
+func TestNoOpClient(t *testing.T) {
+	c := NoOpClient{}
+	tags := []string{"a:b"}
+
+	if c.Gauge("asd", 123.4, tags, 56.0) != nil {
+		t.Error("Gauge output not nil")
```

Comment:
Nit: could you use `github.com/stretchr/testify/assert` (and more specifically `assert.Nil(...)`).

Reply:
Done